src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -0,0 +1,2718 @@
+{
+
+	"death.attack.mmorpg": "%1$s был убит %2$s",
+	"block.mmorpg.harvest.purple": "Хаотические семена",
+	"block.mmorpg.harvest.green": "Первозданные семена",
+	"block.mmorpg.harvest.blue": "Прозрачные семена",
+	"item.mmorpg.custom_loot_crate": "Ящик с добычей",
+	"item.mmorpg.loot_crate": "Ящик с добычей",
+	"item.mmorpg.custom_crate": "Ящик с экипировкой",
+
+"_comment": "# CREATIVE TAB",
+
+	"itemGroup.mmorpg.tab": "Mine and Slash",
+
+	"death.attack.mmorpg.custom_damage": "Много урона!",
+	"death.attack.death_potion": "%1$s подумал, что это отличная идея выпить зелье с названием 'Смерть'.",
+
+"_comment": "STATIONS",
+
+	"block.mmorpg.teleporter": "Картографическое устройство",
```

Сложно, скучно и главное непонятно.
В том же ПоЕ это слово перевели так
```suggestion
	"block.mmorpg.teleporter": "Машина картоходца",
```